roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py

```diff
@@ -50,9 +50,9 @@
 # Examples don't contain auth parameter for simplicity,
 # look at ovirt_auth module to see how to reuse authentication:
 
-# Gather facts about all scheduling policies with name InClusterUpgrade:
+# Gather facts about all scheduling policies with name ClusterInMaintenance:
 - ovirt_scheduling_policies_facts:
-    name: InClusterUpgrade
+    name: ClusterInMaintenance
```

Comment:
But `ClusterInMaintenance` is only in 4.1.4, so what should we use prior to that version? We can't use `ClusterInMaintenance` in <4.1.4.